src/App.tsx

```diff
@@ -15,47 +15,86 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  const visibleGoods = [...goodsFromServer];
+  const [sortType, setSortType] = useState(SortType.NONE);
+  const [isReversed, setIsReversed] = useState(false);
+
+  visibleGoods.sort((a, b): SortType => {
+    switch (sortType) {
+      case SortType.LENGTH:
+        return a.length - b.length;
+
+      case SortType.ALPHABET:
+        return a.localeCompare(b);
+
+      default:
+        return 0;
+    }
+  });
+
+  if (isReversed) {
+    visibleGoods.reverse();
+  }
+
   return (
     <div className="section content">
       <div className="buttons">
         <button
           type="button"
-          className="button is-info is-light"
+          className={`button is-info ${sortType !== SortType.ALPHABET ? 'is-light' : ''}`}
```

Comment:
just a suggestion: let's use [classnames](https://www.npmjs.com/package/classnames) here
using ternary and returning an empty string is not the best approach here